hack/validate-release.sh

```diff
@@ -4,7 +4,6 @@ set -e
 set -u
 
 SUBSCRIPTION="./test/operator-hub/subscription2.yaml"
-BUNDLE_VERSION=`curl -s https://raw.githubusercontent.com/operator-framework/community-operators/master/community-operators/service-binding-operator/service-binding-operator.package.yaml | ./out/venv3/bin/yq -r '.channels[] | select (.name == '\"$CHANNEL\"') | .currentCSV | sub("service-binding-operator.v"; "") '`
```

Comment:
this line was there on purpose: we need to find what is the latest available version available on community-operator. This is not necessary the latest dev release. Thus this need to reverted. As I mentioned in #491 - I cannot reproduce the issue.